package.json

```diff
@@ -67,7 +67,7 @@
     "webpack": "^5.91.0"
   },
   "engines": {
-    "node": "^18.16"
+    "node": "^20.10"
```

Comment:
need to update vercel too

Reply:
it should already be set there: 
<img width="864" alt="image" src="https://github.com/user-attachments/assets/f0af8c80-d0a4-46fa-b902-45410a8b3b8b">

Reply:
Apparently not from the build failure